app/src/main/res/layout/recent_searches.xml

```diff
@@ -19,7 +19,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
+    android:layout_height="match_parent"
```

Comment:
Add this fillViewport to the nested scroll view so the recyclerview doesn't get cut off during the item deletion animation
```
android:fillViewport="true"
```